rpcutils/marshal.go

```diff
@@ -689,3 +692,84 @@ func NewAddAssetSellOrderResponse(
 			"type: %T", e)
 	}
 }
+
+// IsAssetBtc is a helper function that returns true if the given asset
+// specifier represents BTC, and false otherwise.
+func IsAssetBtc(assetSpecifier *priceoraclerpc.AssetSpecifier) bool {
```

Comment:
Since this method doesn't actually pull in any other `taproot-assets` packages, I think we can keep it where it is to address @ffranr's concerns re naming/context.